alacritty/src/cli.rs

```diff
@@ -128,6 +130,12 @@ impl Options {
                     .takes_value(true)
                     .help(&format!("Defines window class on Linux [default: {}]", DEFAULT_NAME)),
             )
+            .arg(
+                Arg::with_name("embed")
+                    .long("embed")
+                    .takes_value(true)
+                    .help("Defines an X window ID (as a decimal integer) which will serve as the parent when embedding Alacritty within other applications."),
```

Comment:
This is rather long still, and doesn't make it super clear that `--embed` is the setting determining that Alacritty **will** be embedded. Maybe something simple like this would be better?

> Defines the X window ID of the parent to embed Alacritty within.

Food for thought.

Reply:
All: I made one more attempt at satisfying everyone's requests here.  It seems like some are in favor of less info, and others in favor of more, I tried to encompass the necessary details everyone has requested in as few words as reasonable.  Hopefully this helps.

@kchibisov Yeah, that was not intentional.  I'm not familiar with this code base and used the most similar other argument I could find as an example of where to add this stuff.  Could you do me a favor and clue me in to what, specifically, is causing this to be bound to the config file?  Is it the presence in src/config/window.rs coupled with some ?  If so, what would be the preferred mechanism for propagating this argument from the command line while bypassing that code path?  The current mechanism seems convenient/clean.

Reply:
Current version seems good to me, though please adjust punctuation accordingly, we want to be consistent here.

I'm very much in favor of short, but it's absolutely necessary to point out the datatype since the default window ID is always printed in the format `0x01238047` as hexadecimal number. And we do not support this.

Reply:
Ok.  Should be consistent now.

Reply:
@mkosem

You should just put `#[serde(skip_deserializing)]` instead of what you're having now iirc in config/window.rs. You're already ignoring this option when you're setting it from cli though, but removing this `+1` config parsing should be a good idea.

Reply:
Thanks! :)  Done.